src/uct/api/uct.h

```diff
@@ -1105,6 +1105,8 @@ UCT_INLINE_API ucs_status_t uct_iface_flush(uct_iface_h iface, unsigned flags,
  * @param [in]    flags  Flags that control ordering semantic (currently
  *                        unsupported - set to 0).
  * @return UCS_OK         - Ordering is inserted.
+ *         UCS_ERR_NO_RESOURCE - Ordering cannot be inserted when the routine
+ *                               is called.
```

Comment:
The meaning of "UCS_ERR_NO_RESOURCE" is a bit tricky. I would suggest to be a bit more descriptive. 
The fence operations can not be posted. Typically, this is an indication of NNN.